ansible/roles/testenv-save-conf/templates/testenv-conf.yaml.j2

```diff
@@ -28,5 +28,6 @@ testbeds:
 system:
   adapterName: Ethernet1
   vHostName: vEthernet (HNSTransparent)
+  mgmtAdapterName: Ethernet0
```

Comment:
what do you think about also renaming `adapterName` to `dataControlAdapterName`?

Reply:
this could be a separate change